src/vivarium_gates_nutrition_optimization/constants/metadata.py

```diff
@@ -16,6 +16,9 @@
 MAKE_ARTIFACT_SLEEP = 10
 
 LOCATIONS = [
+    "Ethiopia",
+    "Nigeria",
+    "Pakistan"
     # TODO - project locations here
```

Comment:
Go ahead and delete the TODOs as appropriate

Reply:
done